### ISSUE TITLE
IOS Padding Still Exists

### ISSUE THREAD

**Issue by CarterBland** [xcarpentier/rn-pdf-reader-js#38]
I am using v0.2.1 and it seems that this error hasn't been fixed. I am attaching a screenshot and the 
code.

https://gyazo.com/6b2bbf4056e9c6b1daad2e7e6bb4d6cb

render function dumbed down
```js
    render () {
      return (
        <LoggedIn title='Recruiting Calendars' {...this.props} hideBack={true} onBack={this.state.current !== null ? () => this.backToList() : undefined}>
          <View style={style.horizontalContainer}>
            <Animatable.View style={style.horizontalContainerChild} ref={'_slide1'}>
              <KeyboardAwareScrollView bounces={false}>
                <Calendars />
              </KeyboardAwareScrollView>
            </Animatable.View>
            <Animatable.View style={style.horizontalContainerChild} ref={'_slide2'}>
              {this.state.current !== null &&
                <PDFReader
                  source={{ uri: this.state.current !== null ? this.state.current.file.data.url : null }}
                />
              }
            </Animatable.View>
          </View>
        </LoggedIn>
      )
    }
```

styles
```js
const style = StyleSheet.create({
  horizontalContainer: {
    flex: 1,
    flexDirection: 'row',
    margin: 0,
    padding: 0
  },
  horizontalContainerChild: {
    flex: 0,
    width: width,
    padding: 0,
    margin: 0
  },
  calendar: {
    backgroundColor: Colors.white,
    padding: 20,
    paddingTop: 20,
    paddingBottom: 20,
    borderBottomColor: 'rgba(0,0,0,0.1)'
  },
  calendarText: {
    color: Colors.black,
    fontFamily: Fonts.header,
    fontSize: 20,
    marginBottom: -10,
    textTransform: 'uppercase'
  },
})
```

**Comment by xcarpentier** [comment]
Can you test with master and? You can use the new `style` props to override some style. Let me know if that help...

**Comment by CarterBland** [comment]
@xcarpentier Alright, that worked for me.

**Comment by vinitsood** [comment]
Is this published on npm yet? Just installed "^0.2.1" via npm but I can't pass style as a prop. The code in **node_modules** for rn-pdf-reader-js is the following:

```
render() {
    const { ready, data, ios, android } = this.state

    if (ready && data && ios) {
      return (
        <View style={styles.container}>
          <WebView
            originWhitelist={['http://*', 'https://*', 'file://*', 'data:*']}
            style={styles.webview}
            source={{ uri: data }}
          />
        </View>
      )
    }
    ... 
```



but in master i see that render is: 
```
 render() {
    const { ready, data, ios, android } = this.state
    const { style } = this.props

    if (ready && data && ios) {
      return (
        <View style={[styles.container, style]}>
          <WebView
            originWhitelist={['http://*', 'https://*', 'file://*', 'data:*']}
            style={styles.webview}
            source={{ uri: data }}
          />
        </View>
      )
    }
```

Thanks! :)
